rules/data-preprocessing.smk

```diff
@@ -64,9 +64,11 @@ rule administrative_borders_gadm:
 rule raw_nuts_units_zipped:
     message: "Download units as zip."
     output:
-        protected("data/automatic/raw-nuts-units.zip")
+        protected("data/automatic/raw-nuts{}-units.zip".format(config["parameters"]["nuts-year"]))
+    params:
+        url = URL_NUTS.format(config["parameters"]["nuts-year"])
```

Comment:
It probably would make sense to verify the parameter value here (within [2006, 2010, 2013, 2016]). `params` accept functions in which you could do that.

Reply:
Would this be better done with a schema to validate against (https://snakemake.readthedocs.io/en/stable/snakefiles/configuration.html#validation)?

Reply:
Absolutely, yes! I wasn't aware this existed.

Reply:
OK, probably a good idea to do this separately. I'll start an issue for it.

Reply:
Hmm you could also start it here and then we extend it later on.

Reply:
Damn, thought you might say that ;)